test/e2e/adminapi_cluster_get.go

```diff
@@ -0,0 +1,38 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/Azure/ARO-RP/pkg/api/admin"
+)
+
+var _ = Describe("[Admin API] Get cluster action", func() {
+	BeforeEach(skipIfNotInDevelopmentEnv)
+
+	It("should be able to return single cluster with admin fields", func() {
+		ctx := context.Background()
+		resourceID := resourceIDFromEnv()
+
+		By("requesting the cluster document via RP admin API")
+		var oc admin.OpenShiftCluster
+		resp, err := adminRequest(ctx, http.MethodGet, resourceID, nil, nil, &oc)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(resp.StatusCode).To(Equal(http.StatusOK))
```

Comment:
duplicates getCluster

Reply:
Good catch! I'll update https://github.com/Azure/ARO-RP/pull/869 or createa a small follow-up to get rid of duplication.